src/PassportServiceProvider.php

```diff
@@ -308,9 +308,12 @@ protected function registerJWTParser()
     protected function registerResourceServer()
     {
         $this->app->singleton(ResourceServer::class, function ($container) {
+            $AccessTokenRepository = $container->make(Bridge\AccessTokenRepository::class);
```

Comment:
Might want to start with a lowercase char for variable names.

```suggestion
            $accessTokenRepository = $container->make(Bridge\AccessTokenRepository::class);
```

Reply:
Done !